src/main/java/org/isf/lab/model/Laboratory.java

```diff
@@ -270,7 +249,7 @@ public int hashCode() {
 	@Override
 	public String toString() {
 		return "-------------------------------------------\nLaboratory{" + "code=" + code + ", material=" + material
-				+ ", exam=" + exam + ", registrationDate=" + labDate + ", examDate=" + examDate + ", result="
+				+ ", exam=" + exam + ", registrationDate=" + labDate + ", examDate=" + createdDate + ", result="
```

Comment:
this should be `labDate`, simply

Reply:
Fixed.

Reply:
Thanks!

Reply:
sorry, spotted now... `registrationDate=" + createdDate + ", examDate=" + labDate`